pkg/integration/components/runner.go

```diff
@@ -210,7 +216,7 @@ func getLazygitCommand(
 	// lazygit requires that certain environment variables like $PATH
 	// be set. TestEnvironment() provides a restricted passthrough
```

Comment:
I noticed that this comment was out of date. I pushed a simplified version, given that the rationale here is explained more fully in env.go now.